docs/azmcp-commands.md

```diff
@@ -518,6 +518,15 @@ azmcp extension az --command "storage account show --name <account-name> --resou
 azmcp extension az --command "vm list --resource-group <resource-group>"
 ```
 
+### Azure Quick Review CLI Extension Operations
+```base
+# Scan a subscription for recommendations
+azmcp extension azqr --subscription-id "<subscriptionId>"
```

Comment:
Use --subscription and resolve it to subscription name or id - see other commands for reference

Reply:
I updated the code to use the global option. However, when I tested the code, the input schema exposed by the MCP server for the azqr tool doesn't include the subscription and the resource group parameters. It only includes the following "tenant", "auth-method", "retry-delay", "retry-max-delay", "retry-mode", "retry-netwaork-timeout". Could you help me figure out what I am missing?